msal/application.py

```diff
@@ -50,6 +50,17 @@ def decorate_scope(
     return list(decorated)
 
 
+def extract_certs(public_cert_content):
+    # Parses raw public certificate file contents and returns a list of strings
+    # Usage: headers = {"x5c": extract_certs(open("my_cert.pem").read())}
+    public_certificates = re.findall(
+        r'\-+BEGIN CERTIFICATE.+\-+(?P<cert_value>[^-]+)\-+END CERTIFICATE.+\-+',
+        public_cert_content, re.I)
+    if len(public_certificates):
```

Comment:
Given that the return value is already a list, possibly empty, the pythonic way to detect a non-empty list is just `if public_certificates:`.